resources/overlay/usr/local/bin/fast_page_fault_helper.c

```diff
@@ -20,18 +20,17 @@
 int main(int argc, char *const argv[]) {
     sigset_t set;
     int signal;
+    void *ptr;
 
     sigemptyset(&set);
-    if(sigaddset(&set, SIGUSR1) == -1) {
+    if (sigaddset(&set, SIGUSR1) == -1) {
         perror("sigaddset");
         return -1;
     }
 
-    void *ptr = mmap(NULL, MEM_SIZE_MIB, PROT_READ | PROT_WRITE, MAP_ANONYMOUS | MAP_PRIVATE, -1, 0);
+    ptr = mmap(NULL, MEM_SIZE_MIB, PROT_READ | PROT_WRITE, MAP_ANONYMOUS | MAP_PRIVATE, -1, 0);
 
-    memset(ptr, 1, MEM_SIZE_MIB);
```

Comment:
I see you add `memset` back in the next commit. Let's do this move in it as well.

Reply:
Argh, yeah, messed this up when distributing the diff across different commits. Fixed!